.github/workflows/gh-pages.yml

```
@@ -204,34 +204,36 @@ jobs:
+      - name: Cache Woff2 output
+        id: cache-woff2
+        uses: actions/cache@6849a6489940f00c2f30c0fb92c6274307ccb58a # v4
+        with:
+          path: fonts/woff2
+          key: python-${{ runner.os }}-fonts-${{ hashFiles('fonts/**') }}
+          restore-keys: |
+            python-${{ runner.os }}-fonts-
+
       - name: Install Poetry
+        if: steps.cache-woff2.outputs.cache-hit != 'true'
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:28: Double quote to prevent globbing and word splitting

(shellcheck)